class-gravity-flow.php

```diff
@@ -6506,6 +6534,23 @@ public function get_user_roles( $user_id = null ) {
 			return (array) $user->roles;
 		}
 
+		/**
+		 * Return the inbox entries count from transient.
+		 * 
+		 * @since 2.5.12
+		 * 
+		 * @return int
+		 */		
+		public function get_workflow_count() {
```

Comment:
Would this make more sense?
```suggestion
		public function get_inbox_count() {
```